README.md

```diff
@@ -80,20 +80,24 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 |
-|---------------------|----------------|----------------|----------------|----------------|
-| client-go 1.4       | +              | ✓              | -              | -              |
-| client-go 1.5       | +              | +              | -              | -              |
-| client-go 2.0       | +              | +              | ✓              | -              |
-| client-go 3.0 beta  | +              | +              | +              | ✓              |
-| client-go HEAD      | +              | +              | +              | +              |
+|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|---------------------|----------------|----------------|----------------|----------------|----------------|
+| client-go 1.4       | +              | ✓              | -              | -              | -              |
+| client-go 1.5       | +              | +              | -              | -              | -              |
+| client-go 2.0       | †              | †              | ✓              | -              | -              |
+| client-go 3.0       | †              | †              | †              | ✓              | -              |
+| client-go HEAD      | †              | †              | †              | +              | +              |
 
 Key:
 
 * `✓` Exactly the same features / API objects in both client-go and the Kubernetes
   version.
 * `+` client-go has features or api objects that may not be present in the
-  Kubernetes cluster, but everything they have in common will work.
+  Kubernetes cluster, but everything they have in common will work. Please
+  note that alpha APIs may vanish or change significantly in a single release.
+* `†` client-go has new features or api objects, and some APIs running in the
```

Comment:
There may not be a thing we actually deprecated, just it is possible that such a thing may exist.